app/models/attendance.rb

```diff
@@ -0,0 +1,11 @@
+class Attendance < ApplicationRecord
+  belongs_to :user
+  belongs_to :school_week
+
+  validates :user, presence: true
+  validates :school_week, presence: true
+  validates :school_period_id, presence: true, numericality: {only_integer: true}
```

Comment:
This reads like it's a foreign key.  Do we have a `SchoolPeriod` class, or does this represent something else?

Reply:
Looks like there are `SchoolPeriod`s in the original application, but I'm not sure what they represent yet.